### ISSUE TITLE
password store module not installable

### ISSUE THREAD

**Issue by kouk** [Pext/Pext#291]
**Describe the bug**

**To Reproduce**
1. Go to 'Modules' -> 'Install module' -> 'From online module list'
2. Click on the second dropdown and select 'Password Store'
3. Click OK
4. After the bubble message informs you that the module's dependencies are being installed, nothing happens.

**Expected behavior**
The module get's installed.

**Desktop (please complete the following information):**
 - OS: Ubuntu 19.10
 - Pext version 0.29 (AppImage)

**Additional context**
The console logs contain the following:
```
    ERROR: Command errored out with exit status 1:
     command: /tmp/.mount_PextJEht0o/usr/bin/python -c 'import sys, setuptools, tokenize; sys
.argv[0] = '"'"'/tmp/pip-install-isxeps41/pypass/setup.py'"'"'; __file__='"'"'/tmp/pip-instal
l-isxeps41/pypass/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.re
ad().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"
'"'))' --no-user-cfg egg_info --egg-base /tmp/pip-install-isxeps41/pypass/pip-egg-info
         cwd: /tmp/pip-install-isxeps41/pypass/                
    Complete output (3 lines):                                                               
    Traceback (most recent call last):
      File "<string>", line 1, in <module>                                                   
    ModuleNotFoundError: No module named 'setuptools'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for fu
ll command output.                                                                           
Pext                                                                                         Failed to download dependencies for Password Store                                           Collecting dulwich==0.19.15                                                                    Using cached dulwich-0.19.15-py3-none-any.whl (419 kB)                                     Collecting paramiko==2.7.1                                                                   
  Using cached paramiko-2.7.1-py2.py3-none-any.whl (206 kB)
Collecting pypass==0.2.1      
  Using cached pypass-0.2.1.tar.gz (34 kB)
```

Seems like setuptools was removed here as "unecessary":

https://github.com/Pext/Pext/commit/2cbaec786047c64f806a6a8c752429a59acc59a9

**Comment by TheLastProject** [comment]
After readding setuptools to requirements.txt in cc2bae3daada22e68c78812f9212e675cd7e4c32 and letting @kouk test, it is sadly confirmed that this is not the right fix and that the "unnecessary" removal was more "removed because it doesn't change anything". At least that's one possible solution investigated, not sure yet how to actually fix this.

**Comment by TheLastProject** [comment]
Fixed in efc06b72229500b9aef9dbc7a9138c26a567dfb4, thank you so much for your report!
